.github/workflows/ci.yml

```diff
@@ -134,7 +135,7 @@ jobs:
       if: ${{ matrix.jobtype == 5 }}
       shell: bash
       run: |
-        ./sbt -v "++$SCALA_213!; test;"
+        ./sbt -v "++$SCALA_213!; test; ++$SCALA_3!; all utilControl/test utilRelation/test utilPosition/test"
```

Comment:
Should we create another new job type `10`? 🤔

Reply:
job type 5 took 8 min (the longest one takes 23 min) so I think this is fine.